wiki/Help_Centre/Gameplay/en.md

```diff
@@ -0,0 +1,283 @@
+
+This can be frustrating if you play on an unstable connection. Although the game will try its best to resubmit your score for as long as you leave the client open.
+
+### Is osu! allowed in your local firewall or anti-virus suite?
+
+**Some firewalls or anti-virus suites can block osu! from accessing the Internet, which will prevent it from sending scores.**
+
+Check your software of choice to make sure that `osu!.exe` file is allowed to access the Internet.
+
+### Are you running lots of programs in the background?
+
+**Some programs can interfere with your ability to submit scores.**
+
+Try closing any extra programs before playing osu! if you find that your scores aren't submitting, and that you are not having issues with connectivity otherwise.
+
+### Are the score submission servers up?
+
+**The answer is almost always yes, but check the [osu! server status](https://status.ppy.sh) or the [@osustatus Twitter](https://twitter.com/osustatus) to make sure everything is running smoothly on our end.**
+
+If the servers are offline, do not close osu! until we've fixed the issues and the servers come back up, as the game will try to submit your score again for up to an hour before it gives up.
+
+## Can you upload a score I set for me?
+
+**Unfortunately, no, we cannot upload scores upon request.**
+
+If you were denied a score due to technical reasons or whatever else, the best you can hope for is to repeat your display of skill and achieve it again. Sorry!
+
+## The notes seem out of sync or out of time with the music!
+
+**This can be due to sound processing software running on your machine (e.g., *Razer Surround Audio*). It could also be due to your `Universal Offset` being set incorrectly.**
+
+*Caution: the use of bluetooth audio devices tend to increase audio latency significantly, which could be causing your audio to lag behind.*
+
+Try closing any extra programs which alter or influence the sound coming out of your computer. If this doesn't help, you could try to adjust your `Universal Offset`.
+
+*Notice: you should make sure you are using headphones/earbuds as opposed to speakers when using the `Offset wizard` to get the best experience possible.*
+
+Every individual hardware configuration has a particular `Universal Offset` that determines whether you hear the game's audio a little later or earlier. The default values work for most people, but if you find it isn't for you, you can change it by following three simple steps:
+
+1. Go to the `Options` menu and find the `Universal offset` slider under the `Offset Adjustment` header. The default value is 0; you should only change it if needed.
+2. If the default value (0) isn't working for you, try using the `Offset wizard` button to walk you through setting a new one.
+3. If this still doesn't help, you may want to try updating your sound card or motherboard audio drivers.\
+
+## I'm experiencing lag spikes or stuttering during play!
```

Comment:
Might make sense to link to https://osu.ppy.sh/help/wiki/Performance_Troubleshooting somewhere in this section.